portfolio/src/main/webapp/json.js

```diff
@@ -0,0 +1,12 @@
+// Function that receives the facts as a JSON from the servlet and display them at the webpage.
+async function getFacts() {
+    const getResponse = await fetch('/facts');
+    const facts = await getResponse.json();
+
+    const index = Math.floor(Math.random() * 3);
```

Comment:
At a high level, it seems like either the server should choose a random fact and send it each time, or the client should get the list of facts once, then pick a random one each time w/out going back to the server.

Is there an advantage to this approach I'm missing?